README.md

```diff
@@ -51,4 +51,4 @@ Learn more about some of Stytch's products used in this example app:
 
 - [Stytch React](https://www.npmjs.com/package/@stytch/stytch-react)
 - [Stytch's node client library](https://www.npmjs.com/package/stytch)
-- [with-iron-session](https://github.com/vvo/next-iron-session)
+- [iron-session](https://github.com/vvo/next-iron-session)
```

Comment:
I think we should make a different paragraph for this, since it's not a Stytch product.